homeassistant/components/linear_garage_door/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "email": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "device_id": "Device ID (leave blank to autogenerate)"
```

Comment:
This seems quite unclear. Either remove it or add a `data_description` explaining when it should be used: https://developers.home-assistant.io/docs/data_entry_flow_index#labels--descriptions